### ISSUE TITLE
Invalid option "__self" when used with @babel/preset-react

### ISSUE THREAD

**Issue by alexandernanberg** [facebook/fbt#197]
## 🐛 Bug Report

Using FBT with `@babel/preset-react` is broken if you enable the new jsx transform and/or the `development` option. 

Compilation fails with

```
Invalid option "__self". Only allowed: project, author, preserveWhitespace, subject, common, doNotExtract
```

I think it should just work OOTB with the preset, many get stuck on this particular issue and 

## To Reproduce

```js
const babel = require('@babel/core');

const code = `
function App() {
  return <div>
    <h1><fbt desc="Greeting">Hello world!</fbt></h1>
  </div>;
}
`;

const opts = {
  filename: 'test.js',
  presets: [['@babel/preset-react', { runtime: 'automatic', development: true }]],
  plugins: [
    'babel-plugin-fbt',
    'babel-plugin-fbt-runtime',
  ],
};

const output = babel.transformSync(code, opts);
```

Adding `extraOptions: { __self: true, __source: true }` to `babel-plugin-fbt` doesn't work for namespaced tags, like `fbt:param` and it throws 

```js
const babel = require('@babel/core');
const fbtImportPlugin = require('./babel-plugin-fbt-import');

const code = `
function App() {
  const name = 'you'
  return <div>
    <h1><fbt desc="Greeting">Hello <fbt:param name="name">{name}</fbt:param>.</fbt></h1>
  </div>;
}
`;

const opts = {
  filename: 'test.js',
  presets: [['@babel/preset-react', { runtime: 'automatic', development: true }]],
  plugins: [
    'babel-plugin-fbt-runtime',
    [
      'babel-plugin-fbt',
      {
        extraOptions: { __self: true, __source: true },
      },
    ],
  ],
};

const output = babel.transformSync(code, opts);
```

throws `Invalid option "__self". Only allowed: number, gender, name`

## Expected behavior

FBT should work OOTB with `@babel/preset-react`

## Link to repo (highly encouraged) or paste

☝️ 

## envinfo

```bash
  System:
    OS: macOS 11.2.1
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 84.38 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.15.5 - ~/.nvm/versions/node/v14.15.5/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 6.14.11 - ~/.nvm/versions/node/v14.15.5/bin/npm
```

**Comment by jrwats** [comment]
Fixed in ecc647d77ca0ce7d23c881fdc9e5fdbe22d6fa69
and published in [`babel-plugin-fbt@0.18.0`](https://www.npmjs.com/package/babel-plugin-fbt/v/0.18.0)

**Comment by mrtnzlml** [comment]
I can confirm it works well. Thank you! 👍
